### ISSUE TITLE
Notes: open a note in view mode by default instead of edit mode

### ISSUE THREAD

**Issue by FlaviusRaducu** [Swatian/nexnotes#25]
**Issue Checklist**
 - [x] I confirm that I have searched the issue tracker
 

**Describe The Feature**
As a user, I would like to open my notes in view mode by default, so that they are nicely rendered for me without having to switch to view mode each time.

**Additional Context**
Hi, great job with the app. I especially like that it does not require unnecessary permissions and that it supports markdown. Personally, I frequently return to my notes to review them and, sometimes edit them. Hence, I'd prefer to have the view mode set as the default, or at least the option to do so.

**App Version**
v1.7 downloaded trough F-Droid

**Comment by mmarif4u** [comment]
I am thinking of providing an option in the settings to enable/disable it. This way, current users won’t be affected, and those who want to enable it can do so from the settings.

**Comment by FlaviusRaducu** [comment]
Yes, having a setting to switch between the two behaviours would be the best option for everyone.
